index.html

```diff
@@ -1,12 +1,261 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>ShopUp</title>
-</head>
-<body>
-    ShopUp
-</body>
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="https://fuzz-ui.netlify.app/styles/FuzzUI.css" />
+        <link rel="stylesheet" href="./styles/ShopUp.css" />
+        <title>ShopUp</title>
+    </head>
+    <body>
+        <main class="main-body flex-column p-3">
+            <div class="navigation flex-j-space-a-center p-5">
```

Comment:
You use semantic tag here like nav/header instead of div.

Reply:
Sure thanks